pkg/scheduler/apis/config/v1beta3/default_plugins.go

```diff
@@ -122,17 +67,17 @@ func getDefaultPlugins() *v1beta3.Plugins {
 
 func applyFeatureGates(config *v1beta3.Plugins) {
 	if utilfeature.DefaultFeatureGate.Enabled(features.VolumeCapacityPriority) {
-		config.Score.Enabled = append(config.Score.Enabled, v1beta3.Plugin{Name: names.VolumeBinding, Weight: pointer.Int32Ptr(1)})
+		// Remove VolumeBinding from Score.Disabled if the feature gate is enabled
+		// Since VolumeBinding is the only disabled plugin right now in the default registry, we can just set Disabled to nil
+		config.Score.Disabled = nil
```

Comment:
Answered above, but this is just inverting the current logic since MultiPoint enables every extension by default.

Reply:
it looks like the VolumeBinding plugin already has a no-op scorer if the feature is off, which makes its Score function inert... not sure why we need to mess with the config here based on feature gates

Reply:
@liggitt that's a good point. In that case, I don't know why we were originally doing the logic to modify the default config here. Looks like @ahg-g added that with `applyFeatureGates` in https://github.com/kubernetes/kubernetes/commit/265ef1741f951290f2b302d81c694186b79decfa?

I do think it is beneficial to clearly show that the plugin is disabled, specifically when logging the internal config. Otherwise, it will show as enabled but it's really doing nothing. That sounds weird to me

Reply:
yeah, we can just depend on the logic in the plugin itself.

Reply:
@damemi we want admins to think about features, less about plugins. I am ok with removing the logic here and depend on the one in the plugin itself.

Reply:
Should we do the same for `DefaultPodTopologySpread` and just remove `applyFeatureGates()` entirely?

Reply:
I think we can leave that for now. I'm planing to GA next release. At least there are no partial disables in that case.

Reply:
I don't think a partial or full disable is relevant, we're going to be disabling some plugins silently based on a featuregate and others explicitly through the config. That's a confusing inconsistency

Reply:
I'm less worried about enabling a plugin based on a feature gate, when the feature gate refers to the entire plugin.

In the next release, when the feature graduates to GA, the plugin would disappear from defaults. But the plugin should still be available for use. Can we do that without bumping up the version? I think removing a plugin from the defaults is backwards compatible.

Reply:
I don't think consistency is important in this context.

VolumeBinding scoring is an extension of the plugin, we don't want to encourage thinking about disabling/enabling extensions, we want to shift towards whole feature/plugin model. To make that point, I don't think we should be disabling the extension here based on the feature gate.

Enabling SelectorSpread is different in my opinion because it is about a whole plugin; so module the comment I left there, I think we can keep it as is (enabled/disabled based on the feature gate).

Reply:
Thought about this some more, and I think the distinction @ahg-g and @alculquicondor are making is reasonable. The goal of MultiPoint is to abstract away the specific extension points as much as possible from administrators.

While I do think it is confusing to see a plugin "enabled" for one extension point where it's really a no-op, I am also biased toward thinking in terms of the specific extension points because that's all we have had until this point.

So it appears inconsistent to treat these different plugins differently, but like Abdullah mentioned the difference is in an entire feature, not an extension point.

Going in the other direction for that, there could be features that are subsets of specific extension points. (For example, if Scoring was enabled but there was a gate to tune the scoring method even further). That type of setting doesn't show up in the Component Config either, but it is still valid to say the plugin is enabled. We want to apply the same idea to MultiPoint, where entire plugins are the features.

So, I'll cede this and remove the logic that I had in this function. @Huang-Wei what do you think?

Reply:
My major concern was the ambiguousness it (the inconsistency) may deliver to the developers - they will have to understand the underlying nuances to better contribute to the codebase.

While for administrators, this approach is fine with me. For non-GA features, they use the feature gate to control  its on/off without worrying about whether this feature is part of a multipoint or not; for GAed features, they leverage the `enabled/disabled` directives or tune plugin arguments of multipoint or regular point to turn on/off the feature.